internal/flowrate/util.go

```diff
@@ -29,6 +29,7 @@ func clockToTime(c time.Duration) time.Time {
 
 // clockRound returns d rounded to the nearest clockRate increment.
 func clockRound(d time.Duration) time.Duration {
+	//nolint:durationcheck
```

Comment:
https://github.com/charithe/durationcheck says you can't multiply two durations... But this lib hasn't been touched in ages.